R/generateSettings.R

```diff
@@ -4,10 +4,12 @@
 #'
 #' The function is designed to work with the SDTM and AdAM CDISC(<https://www.cdisc.org/>) standards for clinical trial data. Currently, eDish is the only chart supported.
 #'
-#' @param standard The data standard for which to create settings. Valid options are "SDTM", "AdAM" or "None". Default: \code{"SDTM"}
-#' @param charts The chart or chart(s) for which standards should be generated ("eDish" only for now) . Default: \code{"eDish"}.
+#' @param standard The data standard for which to create settings. Valid options are "SDTM", "AdAM" or "None". Default: \code{"None"}.
+#' @param charts The chart or chart(s) for which settings should be generated. Default: \code{NULL} (uses all available charts).
+#' @param useDefaults Specifies whether default values from settingsMetadata should be included in the settings object. Default: \code{TRUE}. 
```

Comment:
would you ever not use the defaults?

Reply:
Cuz the standards dont have all of the variables so you'd need to set those somehow right?

Reply:
hmm ... I was thinking that there might be a use case where you want a shell with no values filled in, so I was thinking you could call `generateSettings(standard="None", useDefaults=FALSE) ... although I guess you could also just run `generateShell()` directly in that case. 

I guess I'll leave it in since it's not hurting anything, but definitely an edge case.